.github/CODEOWNERS

```diff
@@ -1,92 +1,98 @@
 # See https://help.github.com/articles/about-codeowners/
 # for more info about CODEOWNERS file
 
+########################################################################
+#                   Updating CODEOWNERS file guide
+# - Add new entry under respective DEE team name
+# - Add new entry in alphabatical order under the team name
+# - Add common Github team owners first followed by specific ones
+# - Always add Github teams as owners instead of individual usernames
+# - Ensure that go/jds-codeowners file is up to date
```

Comment:
As I am working on a horizontal view for all languages, can I suggest to use go/dee-codeowners (WIP) so we can have a single source of truth for all DEE owned languages?

Reply:
Changed this for now. But there are multiple directories that we find in this repo - missing in go/dee-codeowners. I think this will prevent users from easily finding the correct OWNERS.